src/include/function/function.h

```diff
@@ -47,10 +47,12 @@ struct ScalarBindFuncInput {
     const binder::expression_vector& arguments;
     Function* definition;
     main::ClientContext* context;
+    std::vector<std::string> optionalParams;
```

Comment:
I would call this optionalArguments for consistency. Also if optionalArguments has 1-1 relationship with arguments let's document it as well.